v3/auth/sts.go

```diff
@@ -65,9 +66,16 @@ func NewSTSAuth(cerberusURL, region string) (*STSAuth, error) {
 		headers: http.Header{
 			"Content-Type": []string{"application/json"},
 		},
+		credentials: creds(),
 	}, nil
 }
 
+//WithCredentials sets credentials for the STSAuth
+func (s *STSAuth) WithCredentials(c *credentials.Credentials) *STSAuth {
```

Comment:
Let's make `a` the variable for `*STSAuth` to keep it the same as the other functions.

Reply:
Done.